Software/Development/SemesterProject/App/positionCalculator.c

```diff
@@ -0,0 +1,112 @@
+/*
+ * PositionCalculator.c
+ *
+ *  Created on: 17 abr. 2019
+ *      Author: Aitor Teran
+ *
+ *      The module gathers information about the position and speed of the rotor as well  as of it's flux.
+ *      The modules encoderManager and slipAngleCalculator are submodules of this and their functions are
+ *      called from this module.
+ *      All information regarding the position of the rotor or its flux is read from this module.
+ *      motorPosSpeedObject includes all the information.
+ *
+ *      Function to call to calculate parameters is getRotorPosSpeedParameters();
+ *      It should be called at switching frequency.
```

Comment:
I think in the header we are missing a description of how is this module going to be used. I think that getRotorPosSpeedParameters() is the higher level function which includes everything and then this is the one to call all the stuff, I think it would be nice to write that (and maybe place it so it's the first function you see)